scholly/base/src/global/data.js

```diff
@@ -12,17 +12,18 @@ import {
   FAKE_TEAM_MEMBERS,
   FAKE_TEAM_HOMEPAGE,
   FAKE_WEB_PAGES,
-  FAKE_TEAM_AWARDS,
+  FAKE_TEAM_ACHIEVEMENTS,
   FAKE_LAYOUT_OPTION,
 } from './fakeData';
 
 const env = process.env; // eslint-disable-line prefer-destructuring
+const defaultLayout = 1;
```

Comment:
nit: add `\n` to separate `const` and `let`.

Reply:
noted!